Index: sys/arch/i386/conf/majors.i386
===================================================================
RCS file: /cvsroot/src/sys/arch/i386/conf/majors.i386,v
retrieving revision 1.43
diff -u -u -r1.43 majors.i386
--- sys/arch/i386/conf/majors.i386      25 Feb 2012 10:55:28 -0000      1.43
+++ sys/arch/i386/conf/majors.i386      13 Jan 2015 02:20:25 -0000
@@ -92,6 +92,7 @@
 device-major   dmoverio        char 92                 dmoverio
 device-major   cgd             char 93  block 21       cgd
 # major 94 free (ex veriexec which is now a MI major)
+device-major   pcd             char 94 block 25        pcd
 device-major   dpt             char 96                 dpt
 device-major   twe             char 97                 twe
 device-major   nsmb            char 98                 nsmb
